File: Backend/api_seguridad/src/main/java/com/imperio/service/model/dto/existencias/ExistenciaDto.java
package com.imperio.service.model.dto.existencias;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ExistenciaDto {
    private Integer idExistencias;
    private Integer stock;
    private Integer cantidad;
    private String estado;
}

File: Backend/venta/venta/DTO/ExistenciaDto.cs
namespace venta.DTO
{
    public class ExistenciaDto
    {
        public int IdExistencias { get; set; }
        public int IdProductos { get; set; }
        public int? Stock { get; set; }
        public int Cantidad { get; set; }
        public string Estado { get; set; }
    }
}
